quickstarts/mssql-queryplan/dashboards/mssql-queryplan.json

```diff
@@ -0,0 +1,608 @@
+              "width": 8
+            },
+            "title": "Open Transactions",
+            "rawConfiguration": {
+              "dataFormatters": [],
+              "facet": {
+                "showOtherSeries": false
+              },
+              "nrqlQueries": [
+                {
+                  "accountId": 0,
+                  "query": "FROM MssqlCustomQuerySample SELECT label.db_hostname, activeProcesses_session_id, activeProcesses_sql_hostname, activeProcesses_last_batch_time, activeProcesses_open_transaction_count, activeProcesses_database_name, activeProcesses_sql_command, activeProcesses_query_text, activeProcesses_program_name, activeProcesses_spid_status, activeProcesses_start_time, activeProcesses_cpu_time, activeProcesses_logical_read_count  WHERE activeProcesses_open_transaction_count > 0 since 2 months ago"
+                }
+              ]
+            },
+            "linkedEntityGuids": null
+          },
+          {
+            "visualization": {
+              "id": "viz.table"
+            },
+            "layout": {
+              "column": 1,
+              "row": 15,
+              "height": 4,
+              "width": 12
+            },
+            "title": "Blocking Queries",
+            "rawConfiguration": {
+              "dataFormatters": [],
+              "facet": {
+                "showOtherSeries": false
+              },
+              "nrqlQueries": [
+                {
+                  "accountId": 0,
+                  "query": "FROM MssqlCustomQuerySample\nSELECT blockingProcesses_sql_hostname, blockingProcesses_resource_type, blockingProcesses_database_name,\nblockingProcesses_block_object, blockingProcesses_request_mode, blockingProcesses_waiter_spid,\nblockingProcesses_waiter_text, blockingProcesses_blocking_spid, blockingProcesses_blocking_lock,\nblockingProcesses_blocking_text\nWHERE blockingProcesses_resource_type IS NOT NULL since 12 months ago\nLIMIT MAX"
+                }
+              ]
+            },
+            "linkedEntityGuids": null
+          },
+          {
+            "visualization": {
+              "id": "viz.table"
+            },
+            "layout": {
+              "column": 1,
+              "row": 19,
+              "height": 3,
+              "width": 12
+            },
+            "title": "Failed Jobs/Backups",
+            "rawConfiguration": {
+              "dataFormatters": [],
+              "facet": {
+                "showOtherSeries": false
+              },
+              "nrqlQueries": [
+                {
+                  "accountId": 0,
+                  "query": "SELECT latest(failedJobs_job_message) FROM MssqlCustomQuerySample WHERE failedJobs_run_status IN (0,4) FACET label.db_hostname, failedJobs_job_name, failedJobs_run_date, failedJobs_run_time SINCE 1 day ago"
+                }
+              ]
+            },
+            "linkedEntityGuids": [
+              "MjgyMjQyMnxWSVp8REFTSEJPQVJEfDIxMzYxMzU"
```

Comment:
I believe you'll want to make this value "null" as this will be imported by users that will have different linkedEntityGuids.

Please change any values to null in the JSON.

Reply:
By making this change, you can resolve this error:

ERROR: The following errors occurred while validating: quickstarts/mssql-queryplan/config.yml
- quickstartMetadata/dashboards/rawConfiguration/pages/widgets/linkedEntityGuids: `linkedEntityGuids` must not be set

Reply:
All `linkedEntityGuids` have been set to null.